packages/analytics/metric-cards/src/types/metric-card.ts

```diff
@@ -1,21 +1,25 @@
+import type { MetricCardType } from '../enums'
 import type { MetricCardSize } from '../constants'
 export interface MetricCardDef {
+  cardType?: MetricCardType
   currentValue: number
   previousValue: number
   increaseIsBad: boolean
   formatValueFn?: (rawValue: number) => string
   formatChangeFn?: (rawValue: number) => string
+  description?: string
   title?: string
   hasError?: boolean
   tooltip?: string
+  trendRange?: string
 }
 
 export interface MetricCardDisplayValue {
   metricValue: string
   metricChange: string
   changePolarity: number
-  icon?: string
-  cardSize?: string
+  trendIcon?: any
```

Comment:
Is it possible to type this as `GenericIcon`?  No worries if not, just checking.